game/src/edit/traffic_signals/picker.rs

```diff
@@ -0,0 +1,112 @@
+use crate::app::App;
+use crate::common::CommonState;
+use crate::edit::TrafficSignalEditor;
+use crate::game::{PopupMsg, State, Transition};
+use crate::helpers::ID;
+use crate::sandbox::gameplay::GameplayMode;
+use ezgui::{
+    hotkey, Btn, Color, Composite, EventCtx, GeomBatch, GfxCtx, HorizontalAlignment, Key, Line,
+    Outcome, VerticalAlignment, Widget,
+};
+use map_model::IntersectionID;
+use std::collections::BTreeSet;
+
+pub struct SignalPicker {
+    members: BTreeSet<IntersectionID>,
+    composite: Composite,
+    mode: GameplayMode,
+}
+
+impl SignalPicker {
+    pub fn new(
+        ctx: &mut EventCtx,
+        members: BTreeSet<IntersectionID>,
+        mode: GameplayMode,
+    ) -> Box<dyn State> {
+        Box::new(SignalPicker {
+            members,
+            composite: Composite::new(Widget::col(vec![
+                Widget::row(vec![
+                    Line("Select multiple traffic signals")
+                        .small_heading()
+                        .draw(ctx),
+                    Btn::text_fg("X")
+                        .build(ctx, "close", hotkey(Key::Escape))
+                        .align_right(),
+                ]),
+                Btn::text_fg("Edit").build_def(ctx, hotkey(Key::E)),
```

Comment:
UX note: seeing "Edit" here confused me. Maybe something like "Continue" would be better? Take it or leave it though.

<img width="844" alt="Screen Shot 2020-08-13 at 2 35 08 PM" src="https://user-images.githubusercontent.com/217057/90190478-ec150700-dd73-11ea-82a1-e41a52ba72f7.png">

Or if we wanted to get really fancy, something like "Edit (3) Intersections" might make it clearer that the "Edit" button takes you "back".

Reply:
Done. Also made the button use the more prominent style. The fancier thing is nice -- we could even disable the button when 0 things are selected -- but I'll leave that as TODO for now.